src/frontend/apps/impress/src/features/docs/doc-editor/components/BlockNoteToolBar/BlockNoteToolbar.tsx

```diff
@@ -56,13 +59,13 @@ export const BlockNoteToolbar = () => {
         {toolbarItems}
 
         {/* Extra button to do some AI powered actions */}
-        <AIGroupButton key="AIButton" />
+        {conf?.AI_FEATURE_ENABLED && <AIGroupButton key="AIButton" />}
 
         {/* Extra button to convert from markdown to json */}
         <MarkdownButton key="customButton" />
       </FormattingToolbar>
     );
-  }, [toolbarItems]);
+  }, [toolbarItems, conf]);
```

Comment:
```suggestion
  }, [toolbarItems, conf?.AI_FEATURE_ENABLED]);
```
Can you confirm @AntoLC ? From my past life as a react developer ^^

Reply:
Yes you'r right, better like that.